pkg/controllers/clusterresourceplacement/controller_test.go

```diff
@@ -1688,24 +1696,6 @@ func TestGetOrCreateClusterResourceSnapshot_failure(t *testing.T) {
 				},
 			},
 		},
-		{
-			// Should never hit this case unless there is a bug in the controller or customers manually modify the clusterResourceSnapshot.
```

Comment:
may i know why we remove this test case?

Reply:
Is this not the same as the one above?

Reply:
oh, you're right, it's duplicated.